tests/integration/test_fmu_single_zone_vav.py

```diff
@@ -0,0 +1,209 @@
+# Author: Nicholas Long / Sourav Dey
+
+import datetime
+import json
+import os
+import pytest
+import sys
+import time
+from unittest import TestCase
+
+from alfalfa_client import AlfalfaClient, Historian
+from alfalfa_worker.lib.thermal_comfort import ThermalComfort
```

Comment:
I'm thinking that ThermalComfort and units should get moved to alfalfa_client, right?

Reply:
I think it could be.  Can leave here for now

Reply:
i'll move soon-ish. I want to get parity with BOPTEST next.